tests/server/orchestration/api/test_workers.py

```diff
@@ -1845,28 +1845,52 @@ async def test_updates_last_polled_on_a_multiple_work_queues(
                 assert work_queue.last_polled is None
 
     async def test_updates_last_polled_on_a_full_work_pool(
```

Comment:
changes to this test adds coverage that all requested work queues' last polled field gets updated regardless of their status which covers my changes to the `polled_work_queue_ids` up in the app code.